.gitlab-ci.yml

```diff
@@ -558,6 +558,7 @@ build-linux-substrate:             &build-binary
   <<:                              *collect-artifacts
   <<:                              *docker-env
   <<:                              *build-refs
```

Comment:
not going to work on every PR. This line should be changed and the `publish-docker-substrate` should publish to a different registry then.